eyelid-match-ops/src/primitives/poly/modular_poly/conf.rs

```diff
@@ -42,9 +42,8 @@ pub trait PolyConf: Copy + Clone + Debug + Eq + PartialEq {
     fn coeff_zero() -> &'static Self::Coeff;
 }
 
-impl PolyConf for IrisBits {
-    // This degree requires a larger modulus, Fq79 doesn't work
-    const MAX_POLY_DEGREE: usize = IrisBits::DATA_BIT_LEN.next_power_of_two();
+impl PolyConf for FullRes {
+    const MAX_POLY_DEGREE: usize = FullBits::BLOCK_AND_PADS_BIT_LEN.next_power_of_two();
```

Comment:
Polynomial sizes are now calculated automatically, by rounding up the block size to the next power of two.